GooglePlayInstant/Editor/PlayInstantQuickDeployConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Contains set of operations for storing and retrieving quick deploy configurations.
+    /// </summary>
+    public static class PlayInstantQuickDeployConfiguration
+    {
+        private static readonly string ConfigurationFilePath =
+            Path.Combine("Library", "PlayInstantQuickDeployConfig.json");
+
+        public static readonly Configuration Config = LoadConfiguration();
+
+        /// <summary>
+        /// Commit the current state of quick deploy configurations to persistent storage.
+        /// </summary>
+        public static void SaveConfiguration()
+        {
+            File.WriteAllText(ConfigurationFilePath, JsonUtility.ToJson(Config));
+        }
+
+        // Loads and returns Configuration representation of the configuration file if it exists, otherwise instantiate
+        // and return an empty Configuration instance.
```

Comment:
I think you can delete this comment since the implementation is fairly self-evident.

Reply:
Resolved.